src/session_impl.cpp

```diff
@@ -1715,6 +1715,7 @@ namespace aux {
 		}
 
 		ret.udp_sock->set_force_proxy(m_settings.get_bool(settings_pack::force_proxy));
+		ret.udp_sock->set_proxy_settings(proxy());
```

Comment:
sure, but let me get the confirmation that this fix the problem first, no need to triggers more CI jobs :)